packages/lablgtk/lablgtk.2.18.13/files/lablgldir.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/configure b/configure
+index 177e9547..f8a56fe0 100755
+--- a/configure
++++ b/configure
+@@ -4925,7 +4925,7 @@ printf %s "checking lablGL directory... " >&6; }
+       cat > conftest.ml << EOF
+       open Raw
+ EOF
+-      if $CAMLC -c -I "${LABLGLDIR:=+lablGL}" conftest.ml > /dev/null 2>&1 ; then
++      if $CAMLC -c -I "$LABLGLDIR" conftest.ml > /dev/null 2>&1 ; then
```

Comment:
Is there a reason why this patch is not upstream?

Reply:
Originally, it was because the original form was more comfortable for non-opam install.
Will think about that for the next release.